CHANGELOG.md

```diff
@@ -15,13 +15,20 @@
 - Minor benchmark script improvements #2768 (@cyqsimon)
 - Update Arch Linux package URL in README files #2779 (@brunobell)
 - Update and improve `zsh` completion, see #2772 (@okapia)
+- More extensible syntax mapping mechanism #2755 (@cyqsimon)
+  - [BREAKING] Precedence order of user-defined syntax mappings (using `-m/--map-syntax`) has been reversed
+    - `-m '*.foo:alpha' -m '*.foo:bravo'` now maps `bar.foo` to `alpha`
```

Comment:
I think I've read through the majority of the communication on this changeset, but I haven't seen any reason for this. Can you remind me why this is necessary? I would personally prefer it to have the old behavior, I think. Typically, later CLI arguments overwrite previous ones.

Reply:
> but I haven't seen any reason for this.

Yeah. This was a conscious decision by me to do so. My reasoning was that it maintains consistency with the ordering of the builtin mappings. Although when I come to think about it, the user doesn't really know or care about the implementation, so this justification is a bit far-fetched 🤔.

Okay yeah you've convinced me. I'll restore the old behaviour.